README.md

```diff
@@ -18,7 +18,7 @@ pip install -r requirements.txt
 ```bash
 flask --app app run --debug --no-reload
 ```
-#### Running in production mode:
-```bash
-TODO
-```
+> [!WARNING]
+> Currently the app won't destroy the Docker container after the user closes the tab.</br>
```

Comment:
Should be reworded. The app does indeed destroy the containers after INSTANCE_LIMIT amount of time is reached. It's just that if you close the flask app and the containers are still running(because possibly the app crashed or you did a Ctrl-c and the instance runtime did not reach the "shutdown limit" yet), then even if the flask app closes it does not shutdown the currently running containers. 

Tabs currently don't correspond to closing containers because for example:
- User launches new instance (with INSTANCE_LIMIT = 1 (minute) )
- A server-side session is created for 1 min.
- A container is created for 1 min.
- While the session exists, the container exists. User can refresh, or even close and reopen tabs. Because the cookie for the session exists in user's browser, the session runs till 1 min is reached. Similar story for the container.

@TretornESP  should we change this behavior? Should refreshing page or closing the tab be counted as shutting down the container? On one hand, it means more available containers per server. On the other hand, it means too many instance launching and shutdowns. (Imagine someone spamming refresh)

Reply:
Noted, I will keep the conversation open tho so Tretorn can have time to reply here